docs/ecosystem/officials/zio-mock.md

```diff
@@ -476,46 +472,47 @@ object MockPolyService extends Mock[PolyService] {
 
   // We will learn about the compose layer in the next section
   val compose: URLayer[Proxy, PolyService] =
-    ZIO.serviceWithZIO[Proxy] { proxy =>
-      withRuntime[Any].map { rts =>
-        new PolyService {
-          def polyInput[I: Tag](input: I)               = proxy(PolyInput.of[I], input)
-          def polyError[E: Tag](input: Int)             = proxy(PolyError.of[E], input)
-          def polyOutput[A: Tag](input: Int)            = proxy(PolyOutput.of[A], input)
-          def polyAll[I: Tag, E: Tag, A: Tag](input: I) = proxy(PolyAll.of[I, E, A], input)
+    ZLayer(
+      for {
+        proxy <- ZIO.service[Proxy]
+      } yield { 
+          new PolyService {
+            def polyInput[I: Tag](input: I)               = proxy(PolyInput.of[I], input)
+            def polyError[E: Tag](input: Int)             = proxy(PolyError.of[E], input)
+            def polyOutput[A: Tag](input: Int)            = proxy(PolyOutput.of[A], input)
+            def polyAll[I: Tag, E: Tag, A: Tag](input: I) = proxy(PolyAll.of[I, E, A], input)
         }
       }
-    }.toLayer
+    )
 }
 ```
 
 Similarly, we use the same `of` combinator to refer to concrete monomorphic call in our test suite when building expectations:
 
-```scala
+```scala mdoc:silent:nest
 import zio.test._
-import MockPolyService._
```

Comment:
Could keep the import and just do `PolyInput.of[String]`. This would probably be slightly cleaner.